conftest.py

```diff
@@ -23,3 +25,8 @@ def pytest_runtest_logreport(report: pytest.TestReport):
     if report.when == "call" and getattr(report, "context", None):
         if report.failed:
             SUBTESTS_FAILURES[report.nodeid].append(report)
+
+
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
I think it would be preferable to move this fixture into tester.py

Reply:
Doesnt it make sense to have fixture grouped? Tester used the logging but in the essence there is not much binding it to it and fixtures can be used from conftest everywhere

Reply:
There could also be impact and dependency to fixtures order in file.

Reply:
well, since there are already a bunch of fixtures in tester.py it might be better.
especially if there are relatively similar fixtures there, like:
```
    @pytest.fixture(autouse=True, name='setup_logging')
    def fixture_setup_logging(self):
        self._init_logging()
```
i'll move it there as well then.

Reply:
@fruch , seems like this idea [doesn't work well](https://argus.scylladb.com/tests/scylla-cluster-tests/3295da96-8524-4116-896b-b83f11360563). It might somehow has a different order of setup that ends up with sct log missing.
[rerunning](https://argus.scylladb.com/tests/scylla-cluster-tests/2d0f7603-996c-4e38-a50f-025533b9291f) the same where the fixture moved back to conftest did get the sct log collected.